specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/multiple.requests.for.different.realms/request.rpt

```diff
@@ -0,0 +1,20 @@
+connect http://localhost:8000/resource # connecting to two different connections
+connected
+
+write method "GET"
+write header "Authorization" "Basic am9lOndlbGNvbWU=" #login joe:welcome
+write close
+
+read status "401" "Unauthorized"

```

Comment:
If joe:welcome is a valid user and password then according to RFC 7235 (section 2.1) the response should be 403 Unauthorized
```
   A server that receives valid credentials that are not adequate to
   gain access ought to respond with the 403 (Forbidden) status code
   (Section 6.5.3 of [RFC7231]).
```